### ISSUE TITLE
Plural key and default value is not created

### ISSUE THREAD

**Issue by matsgm** [i18next/i18next-parser#212]
- Bug
- 1.0.2?
- Cli

## What I'm trying to do?

* Creating locale files with plural. The default value works when testing without language files and single plural count.

## How am I doing it?

* t('vessel:assistance.attachedMembershipTitle', 'Tilknyttet medlemskap', { count: vessel.memberships.length, defaultValue_plural: 'Tilknyttede medlemskap' })

Here is my config file:

```
module.exports = {
  contextSeparator: '_',
  createOldCatalogs: true,
  defaultNamespace: 'translation',
  defaultValue: '',
  indentation: 2,
  keepRemoved: false,
  keySeparator: '.',
  lineEnding: 'auto',
  locales: ['nb'],
  namespaceSeparator: ':',
  output: 'src/locales/$LOCALE/$NAMESPACE.json',
  input: undefined,
  reactNamespace: false,
  sort: true,
  useKeysAsDefaultValue: false,
  verbose: false,
};

```

The command I'm running:

```
npm run: i18next 'src/**/*.{tsx,ts}'
```

## What was the result?

* Only the "single" default value is created

## What was I expecting?

* The plural default value is also created.

**Comment by karellm** [comment]
I wasn't aware of the `defaultValue_plural` option. A PR with a failing test case is welcome.

**Comment by vincaslt** [comment]
Yeah, haven't seen it as well, the docs don't explicitly show `default_plural` use case at least. Is it the same with numbered plurals like `default_1`, `default_3` and so on?

**Comment by matsgm** [comment]
> Yeah, haven't seen it as well, the docs don't explicitly show `default_plural` use case at least. Is it the same with numbered plurals like `default_1`, `default_3` and so on?

Yes: https://www.i18next.com/translation-function/essentials#overview-options

**Comment by karellm** [comment]
Plural need some love apparently. It might be a good time to look at #169 too. @vincaslt do you have the bandwidth for this?

**Comment by Yuripetusko** [comment]
Bumping this. Running parser creates same value for singular and plural version without respecting `defaultValue_plural`. And if we then run locize sync - we will overwrite locize value with singular form

**Comment by crazyyi** [comment]
I can't get it working with `defaultValue_plural`
To put plural default values in json files, I have to add a key called 'defaultValue_other' in t function, like this:
```
{t("home:itemCount", "{{ count }} unit", {
                    count: v.count,
                    defaultValue_other: "{{ count }} units"
                  })}
```
You can see my example here: https://github.com/crazyyi/next-i18next-working-example
